proto/build-container/Dockerfile

```diff
@@ -53,6 +53,6 @@ RUN PROTOC_VERSION=3.20.1 \
     PROTOC_GEN_GO_VERSION=1.5.2 \
     NODEJS_GRPC_VERSION=1.24.2 \
     NODEJS_GRPC_TOOLS_VERSION=1.8.1 \
-    PYTHON_GRPCIO_VERSION=1.49.1 \
-    PYTHON_GRPCIO_TOOLS_VERSION=1.49.1 \
+    PYTHON_GRPCIO_VERSION=1.50.0 \
+    PYTHON_GRPCIO_TOOLS_VERSION=1.50.0 \
```

Comment:
Were these changes **necessary** for Python 3.11?

gRPC is very fragile, so I'm cautious here in case this breaks anything.

Reply:
Without this, we would be building the grpcio wheel from source.  Bumping the version here allows us to use prebuild wheels for 3.11